### ISSUE TITLE
Feature request: CNO

### ISSUE THREAD

**Issue by samuelburbulla** [aai-institute/continuity#56]
Implement Convolutional Neural Operators:

- https://arxiv.org/abs/2302.01178
- https://github.com/bogdanraonic3/ConvolutionalNeuralOperator

In this task, we could also implement standard U-Nets as (discretization-dependent) neural operators.

**Comment by JakobEliasWagner** [comment]
@samuelburbulla does #105 solve this issue?

**Comment by samuelburbulla** [comment]
No, #105 adds a classical CNN as (discretization-dependent) "neural operator", it's not the CNO from the paper.
